rotkehlchen/types.py

```diff
@@ -507,3 +507,18 @@ class EnsMapping(NamedTuple):
 class CostBasisMethod(SerializableEnumMixin):
     FIFO = 1
     LIFO = 2
+
+
+class AddressbookContact(NamedTuple):
```

Comment:
Lol. Actually in the beginning I called it `AddressbookEntry` too :smile:. But then decided to call it contact since variable `contact` means more than `addressbook_entry` and is shorter. But `AddressbookEntry` might be a better name indeed